css/main.css

```diff
@@ -475,73 +350,48 @@ article p {
     font-size:1em;
     color:#878787;
     font-weight:normal;
+    line-height:1.5em;
 }
 
-.ingredients {
-
-    font-family:'Patrick Hand', cursive;
-    font-size:1.25em;
-    font-weight:lighter;
-    color:#939393;
-    margin:30px 0 0 0;
-    height:325px;
-    width:100%;
-
+.ruled-paper {
+    height:300px;
     background-image: url(../img/list-bg.png);
-    background-repeat: repeat-y;
-    background-size: 100%;
-    /*background-color: lightblue;*/
-
-    -webkit-column-count: 2; /* Chrome, Safari, Opera */
-    -moz-column-count: 2; /* Firefox */
-    column-count: 2;
-
-    -webkit-column-gap: 50px; /* Chrome, Safari, Opera */
-    -moz-column-gap: 50px; /* Firefox */
-    column-gap: 50px;
-
-    -webkit-column-width: 100px; /* Chrome, Safari, Opera */
-    column-width: 100px;
-
+    background-repeat:no-repeat;
+    width:100%;
+    background-size: cover;
 }
 
 .ingredients-left-side {
+    font-family:'Patrick Hand', cursive;
+    font-size:1.40em;
+    font-weight:lighter;
+    color:#939393;
     margin:40px 0 0px 75px;
-    height:325px;
-    line-height: 1.65em;
+    /*height:325px; */
+    line-height: 1.50em;
 }
 
 .ingredients-right-side {
-    margin:0px 0 0px 0px;
-    height:325px;
-    line-height: 1.65em;
+    font-family:'Patrick Hand', cursive;
+    font-size:1.40em;
+    font-weight:lighter;
+    color:#939393;
+    margin:40px 0 0px 0px;
+    /*height:325px;*/
+    line-height: 1.50em;
 }
```

Comment:
This is an opportunity for re-factoring--make a single .ingredients class and put the repeating code inside of it. Then, use another class or style based on :first-of-type / :nth-of-type / :last-of-type for individual styling